File: 1 - Exercicios_23.04/Ruby/exercicio4.rb
puts ("Digite quantos anos você viveu")
anos = gets.to_i

puts ("Digite quantos meses você viveu")
meses = gets.to_i

puts ("Digite quantos dias você viveu")
dias = gets.to_i

anos_to_dias = (anos * 365)

meses_to_dias = (meses * 30)

puts ("Você já viveu #{anos_to_dias + meses_to_dias + dias} dias")

File: Modulo_1/1 - Exercicios_23.04/Python/exercicio4.py
ano_usuario = int(input ("Digite quantos anos você viveu "))
mes_usuario = int(input ("Digite quantos meses você viveu "))
dia_usuario = int(input ("Digite quantos dias você viveu "))

anos_em_dias = ano_usuario * 365
mes_em_dias = mes_usuario * 30

soma_dias = dia_usuario + anos_em_dias + mes_em_dias

print (f"Você viveu {soma_dias} dias")